docs/pages/versions/unversioned/sdk/filesystem.md

```diff
@@ -297,6 +297,32 @@ Returns an object with the following fields:
 
 - **resumeData (_string_)** -- The string which allows the api to resume a paused download.
 
+### `FileSystem.getContentUriAsync(fileUri)`
+
+Take a `file://` URI and convert it into content URI (`content://`) so that it can be access through other applications outside Expo.
```

Comment:
```suggestion
Take a `file://` URI and convert it into content URI (`content://`) so that it can be accessed by other applications outside of Expo.
```